### DISCUSSION TITLE
Is there a setting to tab to the next autocomplete suggestion?

### DISCUSSION THREAD

**Discussion by Decedis** [zed-industries/zed#7991] [Feature Requests]
I searched in the discussions tab but didn't see any discussion on this. Is there a way to tab to the next suggestion? Currently `tab` and `enter` do the same thing and choose the first item on the autocomplete list.

```json
[
  {
    "context": "Editor",
    "bindings": {
      "tab": "editor::ContextMenuNext",
      "tab-shift": "editor::ContextMenuPrev"
    }
  }
]
```
I've tried using this in my custom keymap file, but the `tab` key is still acting like `enter`, and the `tab-shift` seems to be going to the first item in the autocomplete list.

**Comment by mrnugget** [comment]
Hey! `up/down` and `ctrl-n`/`ctrl-p` work. See here:

https://github.com/zed-industries/zed/blob/2b56c43f2d2172d0a8dcaadd9096280eaa7d2de7/assets/keymaps/default.json#L483-L491

**Reply by rajatfinbox** [reply]
but is there no way to use tab to cycle through autocomplete suggestions?

**Reply by mrnugget** [reply]
You can rebind the keys, sure!
